README.md

```diff
@@ -1,38 +1,38 @@
-[![Skeleton](https://user-images.githubusercontent.com/1509726/199282306-7454adcb-b765-4618-8438-67655a7dee47.png)](https://www.skeleton.dev/)
+[![Shmeleton](https://user-images.githubusercontent.com/1509726/199282306-7454adcb-b765-4618-8438-67655a7dee47.png)](https://www.Shmeleton.dev/)
 
-[![npm version](https://img.shields.io/npm/v/@skeletonlabs/skeleton?logo=npm&color=cb3837)](https://www.npmjs.com/package/@skeletonlabs/skeleton)
+[![npm version](https://img.shields.io/npm/v/@Shmeletonlabs/Shmeleton?logo=npm&color=cb3837)](https://www.npmjs.com/package/@Shmeletonlabs/Shmeleton)
 [![Chat](https://img.shields.io/discord/1003691521280856084?label=chat&logo=discord&color=7289da)](https://discord.gg/EXqV7W8MtY)
-[![license](https://img.shields.io/badge/license-MIT-%23bada55)](https://github.com/skeletonlabs/skeleton/blob/master/LICENSE)
+[![license](https://img.shields.io/badge/license-MIT-%23bada55)](https://github.com/Shmeletonlabs/Shmeleton/blob/master/LICENSE)
 
-# 💀 Skeleton
+# 💀 Shmeleton
```

Comment:
"Shmeleton" does not work fine as a headline imho. Just a comment!!!

Reply:
It is just a comment as the one above, but as it was on CODE  - it somehow can be resolved. I'l do that just now.

Reply:
And another comment that was written in the conversation - should show up in the code view, as it is the same context no?

Reply:
yes. it does, also you can keep commenting on "resolved" issues/comments/reviews or whatever this is.

Reply:
commenting here is just a comment.